packages/@aws-cdk/aws-servicecatalog/lib/private/validation.ts

```diff
@@ -1,3 +1,5 @@
+
```

Comment:
No need to start with an empty line.

Reply:
gonna try to find how to configure es lint to catch this.